src/GradhSph/GradhSph.cpp

```diff
@@ -140,15 +140,10 @@ void GradhSph<ndim, kernelclass>::DeallocateMemory(void)
 //=================================================================================================
 template <int ndim, template<int> class kernelclass>
 int GradhSph<ndim, kernelclass>::ComputeH
- (const int i,                         ///< [in] id of particle
-  const int Nneib,                     ///< [in] No. of potential neighbours
-  const FLOAT hmax,                    ///< [in] Max. h permitted by neib list
-  FLOAT *m,                            ///< [in] Array of neib. masses
-  FLOAT *mu,                           ///< [in] Array of m*u (not needed here)
-  FLOAT *drsqd,                        ///< [in] Array of neib. distances squared
-  FLOAT *gpot,                         ///< [in] Array of neib. grav. potentials
-  SphParticle<ndim> &part,             ///< [inout] Particle i data
-  Nbody<ndim> *nbody)                  ///< [in] Main N-body object
+ (SphParticle<ndim> &part,                                ///< [inout] Particle i data
```

Comment:
Yes, probably a nicer and cleaner way to do this, and makes it easier if we need additional quantities for computing other things without needing to change the interface all the time.